docs/swarm/index.md

```diff
@@ -33,44 +33,45 @@ services. You don't need additional orchestration software to create or manage
 a Swarm.
 
 * **Decentralized design:** Instead of handling differentiation between node
-roles at deployment time, Swarm handles any specialization at runtime. You can
-deploy both kinds of nodes, managers and workers, using the Docker Engine.
-This means you can build an entire Swarm from a single disk image.
+roles at deployment time, the Docker Engine handles any specialization at
+runtime. You can deploy both kinds of nodes, managers and workers, using the
+Docker Engine. This means you can build an entire Swarm from a single disk
+image.
 
-* **Declarative service model:** Swarm uses a declarative syntax to let you
-define the desired state of the various services in your application stack.
-For example, you might describe an application comprised of a web front end
-service with message queueing services and a database backend.
+* **Declarative service model:** Docker Engine uses a declarative syntax to let
+you define the desired state of the various services in your application
+stack. For example, you might describe an application comprised of a web front
+end service with message queueing services and a database backend.
```

Comment:
Let's say "approach" instead of "syntax", since "syntax" implies there's a file format, which currently only exists in experimental.